app/model/commands/PublishAtomCommand.scala

```diff
@@ -50,14 +50,25 @@ case class PublishAtomCommand(id: String)(implicit val previewDataStore: Preview
           case None => Logger.info(s"No active YouTube asset found for atom: ${id}")
         }
 
-        api.updateMetadata(id, UpdatedMetadata(
-          atom.description,
-          Some(atom.tags),
-          atom.youtubeCategoryId,
-          atom.license,
-          atom.privacyStatus,
-          atom.expiryDate,
-          atom.plutoProjectId))
+        atom.activeVersion match {
+          case Some(atomVersion) => {
+
+            val activeAssetId = atom.assets.find(asset => {
+              asset.version == atomVersion
+            }).get.id
+
+            api.updateMetadata(activeAssetId, UpdatedMetadata(
+              atom.title,
+              atom.description,
+              Some(atom.tags),
+              atom.youtubeCategoryId,
+              atom.license,
+              atom.privacyStatus,
+              atom.expiryDate,
+              atom.plutoProjectId))
+          }
+          case None =>
```

Comment:
This would be the case when the atom has no video so we won't have to make any changes to youtube.